pages/general/tooling-and-ecosystem.md

```diff
@@ -3,14 +3,66 @@
 In addition to an API design specification, AEPs also provide an ecosystem of
 tooling to help produce and interact with these APIs.
 
+## Tools
+
+Get started quickly with these essential tools:
+
+**API Definition**
+
+- **[aepc](https://github.com/aep-dev/aepc)** - Generates AEP-compliant RPCs
+from proto messages
+
+**Linting & Validation**
+
+- **[aep-proto-linter](https://github.com/aep-dev/api-linter)** - Lint gRPC
+  definitions for AEP compliance
+- **[aep-openapi-linter](https://github.com/aep-dev/aep-openapi-linter)** -
+  Lint OpenAPI definitions for AEP compliance
+
+**AI**
+
+- **[aep-mcp-server](https://github.com/aep-dev/aep-mcp-server)** - MCP server
+  for AEP-compliant APIs
+- **[aep.dev/llms.txt](https://aep.dev/llms.txt)** - llms.txt for AEP, to
+  provide LLMs with knowledge of AEP guidelines.
+
+**User Interfaces**
+
+- **[aepcli](https://github.com/aep-dev/aepcli)** - Command-line interface for
+  AEP-compliant APIs
+- **[aep-explorer](https://github.com/aep-dev/aep-explorer)** - Interactive web
+  UI for AEP-compliant APIs
+
+**Infrastructure-As-Code and GitOps**
```

Comment:
What do you think about removing the "planned" tools from all these lists?

Then we can put Terraform and aepcli under this list
```suggestion
**Developer Tools**
```